packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -243,14 +243,22 @@ class MultipartUploader {
 
   async #prepareUploadParts (candidates) {
     this.lockedCandidatesForBatch.push(...candidates)
+    let result
 
-    const result = await this.options.prepareUploadParts({
-      key: this.key,
-      uploadId: this.uploadId,
-      partNumbers: candidates.map((index) => index + 1),
-    })
+    try {
+      result = await this.#retryable({
+        attempt: () => this.options.prepareUploadParts({
+          key: this.key,
+          uploadId: this.uploadId,
+          partNumbers: candidates.map((index) => index + 1),
+        }),
+      })
+    } catch (error) {
+      this.#onError(error)
```

Comment:
Why catch this? Shouldn't we let errors propagate up like the error below and like the previous code (`prepareUploadParts` call) did?